### PR TITLE
date fix on add client and minor fixes to css

### PR DESCRIPTION
**Pull request by MeghanBomberger**
nit pick update for making the icons on the view client page more visible in different vision accessibility modes

### CODE REVIEW

File: frontend/add-client/form-inputs/contact-information-inputs.component.tsx [hunk 1]
```diff
@@ -160,8 +160,12 @@ export default React.forwardRef(function ContactInformationInputs(
 });
 
 function getTodayAsString() {
-  const date = new Date().toISOString();
-  return date.slice(0, date.indexOf("T"));
+  const day = new Date().getDate();
+  const year = new Date().getFullYear();
+  const month = new Date().getMonth();
+
+  const date = `${year}-${month > 10 ? "" : "0"}${month + 1}-${day}`;
```
Comment (joeldenning) [inline frontend/add-client/form-inputs/contact-information-inputs.component.tsx]: Is this fixing a bug with time zones or something? Or why the change here?

Comment (MeghanBomberger) [reply frontend/add-client/form-inputs/contact-information-inputs.component.tsx]: I think it may be related, or at least a similar bug. After 5 it was showing as the next day.

Comment (joeldenning) [reply frontend/add-client/form-inputs/contact-information-inputs.component.tsx]: I'd rather use https://github.com/felixge/node-dateformat to format the dates correctly, and also find a solution that we apply to all places where we're using dateformat.

Maybe try fixing it with dateformat?

Comment (joeldenning) [reply frontend/add-client/form-inputs/contact-information-inputs.component.tsx]: `dateFormat(new Date(), "yyyy-mm-dd")`

Comment (MeghanBomberger) [reply frontend/add-client/form-inputs/contact-information-inputs.component.tsx]: resolved in 96-02 fixes

**Comment by joeldenning** [comment]
Duplicate of #120
